cpp/velox/tests/OrcTest.cc

```diff
@@ -0,0 +1,158 @@
+
+#include "benchmarks/BatchStreamIterator.h"
```

Comment:
Sorry for the late review, now we have two options `BUILD_BENCHMARKS` and `BUILD_TESTS`, there can separately build. Now we need to build them at the same time, this is not a good solution. Can you modify the test to remove benchmark link? @zuochunwei

Reply:
OK, could you help me to process this problem? @yiming-yang

Reply:
OK, could you help me to process this problem? @yiming-yang